src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    Animal.alive.push(this);
+  }
+  // static alive() {
+  //   return this.aliveAnimals;
+  // }
```

Comment:
The commented-out static method `alive()` is redundant since the static property `alive` is already being used to track living animals. Consider removing these lines if this method is not needed.